python_modules/dagster/dagster/_core/instance/runs/run_domain.py

```diff
@@ -720,11 +728,29 @@ def get_keys_to_reexecute(
             {key for key in to_reexecute if isinstance(key, AssetCheckKey)},
         )
 
+    def _get_repo_scoped_asset_node(
+        self,
+        asset_graph: "BaseAssetGraph",
```

Comment:
if we are grabbing it off the request context it should be a `RemoteWorkspaceAssetGraph` right?

Reply:
(called by the 'execute_job' public API in python / the execute_in_process API)